tools/bin/ci_check_dependency.py

```diff
@@ -32,14 +34,18 @@ def main():
     all_connectors = get_all_connectors()
 
     # Getting all build.gradle file
-    all_build_gradle_files = get_connectors_gradle_files(all_connectors)
+    build_gradle_files = {}
+    for connector in all_connectors:
+        connector_path = CONNECTORS_PATH + connector + "/"
+        build_gradle_files.update(get_gradle_file_for_path(connector_path))
+    build_gradle_files.update(get_gradle_file_for_path(NORMALIZATION_PATH))
 
     # Try to find dependency in build.gradle file
-    depended_connectors = list(set(get_depended_connectors(changed_modules, all_build_gradle_files)))
+    dependent_modules = list(set(get_depended_connectors(changed_modules, build_gradle_files)))
```

Comment:
```suggestion
    dependent_modules = list(set(get_depended_modules(changed_modules, build_gradle_files)))
```
Don't you want to rename this function, as it's now tracking connectors and normalization?

Reply:
I do! Must have missed that mid-refactor, thanks for the catch 😄